content/en/docs/quickstart.md

```diff
@@ -14,39 +14,33 @@ Before you begin, you need:
 
 ## Create an OCI-compatible registry
 
-{{% alert title="Warning" color="warning" %}}
-The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for development purposes. When using other registries, ensure the registry is compatible with OCI Image specification v1.1.0. Starting with `v1.0.0-rc.1` of `notation`, by default, signatures are stored using [OCI Artifact Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md), which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)).
-{{% /alert %}}
-
-Create and run an OCI-compatible registry on your development computer using Docker and the [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry) container image. The following command creates a registry that is accessible at `localhost:5000`.
+Create and run an OCI-compatible registry on your development computer using the [distribution/distribution](https://github.com/distribution/distribution) with the [image deletion](https://docs.docker.com/registry/spec/api/#deleting-an-image) enabled. The following command creates a registry that is accessible at `localhost:5001`. 
 
 ```console
-docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.4
+docker run -d -p 5001:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true registry
```

Comment:
Why is the "Image Deletion" option important? Is that a typical case that we want users to know about?

Reply:
@iamsamirzon see comment https://github.com/notaryproject/notaryproject.dev/pull/155#pullrequestreview-1398299217

Reply:
@iamsamirzon your comments posted after this PR was merged. @FeynmanZhou  since this is a warning, maybe we can improve it in later release to add it as a note, not in the main command.